backend/codesirius/problems/views/hidden_test.py

```
@@ -124,3 +129,43 @@ def post(self, request, problem_pk):
         except Exception as e:
             logger.error(f"Kafka message production failed: {e}")
             raise ValidationError("Failed to initiate hidden test processing")
+
+
+class HiddenTestDeleteAPIView(APIView):
+    permission_classes = [IsOwner]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)